packages/block-editor/src/components/link-control/index.js

```diff
@@ -265,36 +410,60 @@ function LinkControl( {
 
 		return (
 			<div className="block-editor-link-control__search-results-wrapper">
-				{ isInitialSuggestions ? (
-					SearchResultsLabel
-				) : (
-					<VisuallyHidden>{ SearchResultsLabel }</VisuallyHidden>
-				) }
-
+				{ isInitialSuggestions && SearchResultsLabel }
```

Comment:
Not relevant for here, but: The capitalization of this variable wrongly implies that it's a component. GitHub's syntax highlighting even becomes confused due to it. This should be `searchResultsLabel` if it's an element.

Reply:
Fixed